Sts1CobcSw/Outcome/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-target_link_libraries(Sts1CobcSw_Outcome INTERFACE rodos::rodos)
+# target_link_libraries(Sts1CobcSw_Outcome INTERFACE rodos::rodos)
```

Comment:
I guess this is related to the whole `USE_NO_RODOS` issue. We'll probably need two `Outcome` targets: one with and one without Rodos. The unit tests can then link to the one without Rodos while the rest uses the "normal" version.

Reply:
Readded in https://github.com/SpaceTeam/STS1_COBC_SW/pull/287/commits/b395f28ac8899dddc896183bcb19de1f1184792b